Spanner/tests/System/SpannerTestCase.php

```diff
@@ -27,7 +27,7 @@
 class SpannerTestCase extends SystemTestCase
 {
     const TESTING_PREFIX = 'gcloud_testing_';
-    const INSTANCE_NAME = 'google-cloud-php-system-tests';
+    const INSTANCE_NAME = 'jdp-spanner-devel'; //'google-cloud-php-system-tests';
```

Comment:
Just leaving this as a reminder that this needs to be removed when the time comes.